drivers/python/rethinkdb/_restore.py

```diff
@@ -135,6 +140,8 @@ def do_import(temp_dir, options):
     import_args.extend(["--directory", temp_dir])
     import_args.extend(["--auth", options["auth_key"]])
     import_args.extend(["--clients", str(options["clients"])])
+    import_args.extend(["--shards", str(options["shards"])])
+    import_args.extend(["--replicas", str(options["replicas"])])
```

Comment:
It looks like this will be passing down `--shards None` and `--replicas None` if those parameters use their default value, which will error in the import script options parsing.